netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -79,14 +79,18 @@
    * noticing changes to DNS.
    */
   public static NettyChannelBuilder forAddress(SocketAddress serverAddress) {
```

Comment:
@zhangkun83 Would the user ever want an inet address to be resolved once at startup?  If the answer is no, then I think we should make this change.